aptos-move/mvhashmap/src/versioned_aggregators.rs

```diff
@@ -464,16 +485,16 @@ impl<K: Eq + Hash + Clone + Debug + Copy> VersionedAggregators<K> {
             })
```

Comment:
In the v.read_latest_committed_value, we have a bunch of unreachables that should really become the code errors, so I think we should return appropriate errors there, and uniformize the return type of read_latest_committed_value and read?